api/src/wfl/module/copyfile.clj

```diff
@@ -14,31 +15,37 @@
    :top     "wdl/copyfile.wdl"})
 
 (defn- submit-workflow
-  "Submit WORKFLOW to Cromwell in ENVIRONMENT."
-  [environment workflow labels]
+  "Submit WORKFLOW to Cromwell in ENVIRONMENT with OPTIONS and LABELS."
+  [environment workflow options labels]
   (cromwell/submit-workflow
     environment
     (util/extract-resource (:top workflow-wdl))
     nil
     (-> workflow (select-keys [:src :dst]) (util/prefix-keys pipeline))
-    (util/make-options environment)
+    options
     labels))
 
 (defn add-copyfile-workload!
   "Use transaction TX to add the workload described by WORKLOAD-REQUEST."
   [tx {:keys [items] :as _workload-request}]
-  (let [[uuid table] (all/add-workload-table! tx workflow-wdl _workload-request)
-        inputs (map :inputs items)]
+  (let [default-options   (-> (:cromwell _workload-request)
+                              all/cromwell-environments
+                              first
+                              util/make-options)
+        [uuid table opts] (all/add-workload-table! tx workflow-wdl _workload-request default-options)
+        to-row (fn [item] (assoc (:inputs item)
+                            :workflow_options
+                            (json/write-str (util/deep-merge opts (:workflow_options item)))))]
```

Comment:
Alternative?
```
(-> item 
  (select-keys [:inputs :workflow_options])
  (update :workflow_options #(json/write-str (util/deep-merge opts %)))))
```

Reply:
Good idea! Put this in WGS too

Reply:
I ended up undoing this because I didn't realize that this can't really deal with the input nesting well. I like the `select-keys` approach but what I really want is just "put JSON-ified workflow_options into the map of inputs"

Reply:
I don't understand.

Reply:
nevermind - i misunderstood your code initially.

Reply:
why are workflow_options part of the inputs?

Reply:
This is how the WGS and copyfile databases are. I know you're reworking WGS, but basically what I did was just add a column of text to be the json encoded options. So since this bit of code is just turning the workflow into a row that we can write into the db, we want to put the workflow options right next to the inputs